src/conversion.rs

```diff
@@ -423,6 +446,107 @@ where
     }
 }
 
+// Process top-level `GeoJSON` items, returning a geo_types::GeometryCollection or an Error
+#[cfg(feature = "geo-types")]
```

Comment:
this file (`conversion.rs`) is specified in `lib.rs` to only be compiled if the `geo-types` feature is enabled:

https://github.com/georust/geojson/blob/093bb5345f7cd7e67c6483cc3bece14583ba01ad/src/lib.rs#L222-L223

so i don't _think_ we need to add any `#[cfg(feature = "geo-types")]` lines to this file

Reply:
Fixed!